examples/matrix_t_test.html

```diff
@@ -6,11 +6,12 @@
 
         var Module = {
             onRuntimeInitialized: function () {
-                console.log(Module)
+                console.log(Module);
+                const U8_t = Module.Types.U8_t.value;
+                const C1_t = Module.Types.C1_t.value;
                 // test if the matrix_t is filled with data
                 var data_u8 = new Uint8Array ([23, 20, 12, 24, 212, 220, 120, 46, 78, 92, 35, 12, 120, 120, 120, 120]);
-                //console.log(data_u8);
-                var m_u8 = new Module.matrix_t(2, 2, 0x0100 | 0x01, data_u8);
+                var m_u8 = new Module.matrix_t(4, 4, U8_t | C1_t, data_u8);
                 console.log(m_u8);
                 console.log(m_u8.data);
```

Comment:
this not output the correct data it should be the same as in data_u8 but instead we get a Uint8Array as this:
`Uint8Array [buffer: ArrayBuffer(268435456), byteLength: 0, byteOffset: 0, length: 0, Symbol(Symbol.toStringTag): 'Uint8Array']`
note the size `268435456` probably it create the Buffer before receiving the data and that's the result. But in the grayscale_example there is not this issue. I will open an issue to track down the problem and to solve it.

Reply:
see issue https://github.com/kalwalt/webarkit-jsfeat-cpp/issues/5